draft-pauly-privacypass-auth-scheme.md

```diff
@@ -370,6 +370,16 @@ by another, as shown below.
 ~~~
 {: #fig-replay title="Token Architectural Components"}
 
+Interactive token challenges require clients to obtain matching tokens when challenged,
+rather than presenting a token that was obtained in the past. This means that issuance
+and redemption events will occur at approximately the same time. In particular, if
+a client is challenged for an interactive token at time T1 and then subsequently obtains
+a token at time T2, a colluding issuer and origin can link this to the same client if
+T2 is unique to the client. This linkability is less feasible as the number of issuance
```

Comment:
Isn't it about T1 and T2 being close together, rather than the uniqueness of T2?

Reply:
That's what I mean by uniqueness, i.e., since T1 is definitely unique to a client, then T2 also is, and so (a) T1 and T2 are close or (b) there's so few requests that they don't need to be close in time. (I went with "unique" instead of "close" to capture both scenarios.)

Reply:
That's what I mean by uniqueness, i.e., since T1 is definitely unique to a client, then T2 also is, and so (a) T1 and T2 are close or (b) there's so few requests that they don't need to be close in time. (I went with "unique" instead of "close" to capture both scenarios.)